pkg/capacity/pci.go

```diff
@@ -0,0 +1,273 @@
+package capacity
+
+import (
+	"bufio"
+	"bytes"
+	_ "embed"
+	"fmt"
+	"os"
+	"path/filepath"
+	"regexp"
+	"strconv"
+	"strings"
+)
+
+const (
+	pciDir = "/sys/bus/pci/devices"
+)
+
+type Device struct {
+	ID   uint16
+	Name string
+}
+
+type Vendor struct {
+	ID      uint16
+	Name    string
+	Devices map[uint16]Device
+}
+
+var (
+	//go:embed pci/pci.ids
```

Comment:
How do we keep this file always updated?

Reply:
Good point, i will add a `go:generate` statement that auto-download the file. the `go generate` call still has to be done before building. I will add generate to the CI as well